amethyst_renderer/src/formats/texture.rs

```diff
@@ -241,7 +241,26 @@ where
 #[derive(Clone, Debug)]
 pub struct ImageData {
     /// The raw image data.
-    pub raw: Image<u8>,
+    pub rgba: RgbaImage,
+}
+
+
+fn load_into_rgba8_from_memory(data: Vec<u8>, options: TextureMetadata) -> Result<TextureData> {
+    ::image::load_from_memory(&data)
```

Comment:
We try to avoid this type of crate referencing if we can, and instead import the function.

Reply:
Would the following be the preferred format:
```
    use ::image::load_from_memory;                                                                   |                                                                                                      
    load_from_memory(&data)
```
?

Reply:
yes

Reply:
Yes.

Reply:
Ok, fixing.